classes/componentsupport/forum_component.php

```diff
@@ -280,7 +280,6 @@ public function resolve_module_instance_id($table, $id) {
         $discussions = '{' . $this->type . '_discussions}';
         $poststable = '{' . $this->type . '_posts}';
         $forum = '{' . $this->type . '}';
-        $params = [$id];
```

Comment:
This local variable is unused since its value is overwritten immediately on line 285.